### PR TITLE
Format code

### PR DESCRIPTION
**Pull request by so1ve**
#20

### CODE REVIEW

**Comment by Misaka13514** [comment]
... Where is [yarn.lock](https://github.com/baiyang-lzy/bbg/pull/21/files#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2de)

**Comment by so1ve** [comment]
> ... Where is [yarn.lock](https://github.com/baiyang-lzy/bbg/pull/21/files#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2de)

Sorry I deleted it because I want to use PNPM XD

File: App/blog_source/index.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
   <meta charset="utf-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   <link rel="shortcut icon" href="./favicon.ico" />
-  <title></title>
+  <!-- <title></title> -->
```
[the remark this answers is not in the document]
Comment (so1ve) [reply App/blog_source/index.html]: Hmmm is this necessary

**Comment by Misaka13514** [comment]
close #20

**Pull request merged**